Description: Remove declaration of function g_queue_free_full() which
 conflicts with GLib and isn't used anywhere.
Author: Ilya Barygin <randomaction@ubuntu.com>

--- amanda-3.3.0.orig/common-src/glib-util.h
+++ amanda-3.3.0/common-src/glib-util.h
@@ -72,7 +72,7 @@ void slist_free_full(GSList * list, GDes
 
 /* These functions all take a GLib container, and call free() on all the
  * pointers in the container before free()ing the container itself. */
-void g_queue_free_full(GQueue * queue);
+//void g_queue_free_full(GQueue * queue);
 void g_ptr_array_free_full(GPtrArray * array);
 
 /* g_value_compare() does what you expect. It returns TRUE if and
--- amanda-3.3.0.orig/common-src/glib-util.c
+++ amanda-3.3.0/common-src/glib-util.c
@@ -120,6 +120,7 @@ void slist_free_full(GSList * list, GDes
 }
 #endif
 
+#if 0
 void g_queue_free_full(GQueue * queue) {
     while (!g_queue_is_empty(queue)) {
         gpointer data;
@@ -128,6 +129,7 @@ void g_queue_free_full(GQueue * queue) {
     }
     g_queue_free(queue);
 }
+#endif
 
 void g_ptr_array_free_full(GPtrArray * array) {
     size_t i;
